tests/Data/AugmentedTest.php

```diff
@@ -318,12 +333,10 @@ class Thing
 {
     use ContainsData;
 
-    public function __construct($data)
+    public function __construct($data, $supplements = ['supplemented' => 'supplemented value'])
     {
         $this->data = $data;
-        $this->supplements = [
-            'supplemented' => 'supplemented value',
-        ];
+        $this->supplements = $supplements;
```

Comment:
Can you revert this?

Reply:
Yes

Reply:
Sorry I meant just the constructor. The test you added was fine. The change the constructor didn't make a difference.

Reply:
Haha 😆 np